### DISCUSSION TITLE
Sorbed concentration output

### DISCUSSION THREAD

**Discussion by cneyens** [MODFLOW-ORG/modflow6#2006] [Q&A] [answered]
Is there a way to get the sorbed concentrations as output? E.g. a ucn file with the sorbed concentrations (I think this was possible with MT3D) or observations of the sorbed concentration at selected cells?
Right now, we do this in postprocessing by calculating them from the aqueous concentration and the sorption parameters.

**Comment by christianlangevin** [comment, marked as answer]
Hi @cneyens, at the moment sorbed concentrations are not available, but they should be.  We will add an issue and put this in our todo list (see Issue #2007).

**Reply by christianlangevin** [reply]
With #2013  you can now save sorbate concentrations to a binary file.  Let us know if you run into any issues.  A more thorough set of GWT observations, such as sorbate concentration, is also on the list;  we will add them as time permits.

**Reply by cneyens** [reply]
Thanks Chris, I can get this to work with the latest nightly build. For completeness, will this functionality also become available for the sorbed concentrations in the immobile domain?

**Reply by christianlangevin** [reply]
Yes, we can add it there too.  We are bringing these capabilities in when requested and as time permits.  More thorough observations are also planned for GWT.
